lib/rwj_reporter.rb

```diff
@@ -1,10 +1,8 @@
+require 'settings_file'
+require 'simple_csv_printer'
+require 'track_merged_csv_printer'
 
 class RwjReporter
-  def self.print_reports_for_dates(start_date_str, end_date_str, log_file_dir=nil, output_dir=nil)
-    rr = RwjReporter.new(start_date_str, end_date_str, log_file_dir)
-    rr.print_reports_for_dates(output_dir)
-  end
-
```

Comment:
I thought this premature optimization might be useful, like if they forgot to do a month or needed 6 months or something.  But I'm not emotionally attached to it.